lib/commonConfig.js

```diff
@@ -155,6 +155,7 @@ function processEnvironmentVariables() {
         'IOTA_MULTI_CORE',
```

Comment:
CNR entry needed?

Reply:
...mentioning defaultEntityNameConjunction and its associated env var

Reply:
fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/949/commits/1435a44f80e8d3fe10c672bdc214952f4c31229b